lib/usecases/applicationDetails.js

```diff
@@ -25,19 +25,21 @@ export default async ({ clientGeneratedId }) => {
       status: APPLICATION_STATE[application.application_state_id - 1],
       grantAmountAwarded: application.grant_amount_awarded,
       eligibilityCriteria: {
-        tradingInHackney: booleanToYesNo(application.trading_in_hackney),
-        liableForRates: booleanToYesNo(application.liable_for_rates),
+        tradingInHackney: application.trading_in_hackney,
+        liableForRates: application.liable_for_rates,
       },
       eligibilityCriteriaDetails: {
         businessSizeId: BUSINESS_SIZE[application.business_size_id - 1],
         howManyEmployees: application.how_many_employees,
-        businessCategory: application.business_category,
-        businessSubCategory: application.business_sub_category,
-        businessCustomCategory: application.business_custom_category,
-        tradingOn220320: booleanToYesNo(application.trading_on_22_03_20),
-        tradingOn161020: booleanToYesNo(application.trading_on_16_10_20),
-        tradingOn041120: booleanToYesNo(application.trading_on_04_11_20),
-        servedLegalNotices: booleanToYesNo(application.served_legal_notices),
+        businessCategory: {
+          business_category: application.business_category,
+          business_sub_category: application.business_sub_category,
+          business_custom_category: application.business_custom_category,
```

Comment:
We are mixing casing here, so I think we should keep them consistent.

Reply:
I am just representing what the user entered

Reply:
The fields on the left are snake cases, whereas the other properties on the left are camelCased. I was wondering if it could be something like the following suggestion instead.

```suggestion
          businessCategory: application.business_category,
          businessSubCategory: application.business_sub_category,
          businessCustomCategory: application.business_custom_category,
```

Reply:
Ahhh sorry I thought you meant UI wise, because of that I was confused 🤦 

<img width="930" alt="Screenshot 2020-11-27 at 09 57 31" src="https://user-images.githubusercontent.com/435399/100441277-7fb1df80-30a6-11eb-87d1-ac5cc9f7e635.png">

Yes, it makes sense, we are not using those fields in the FE, we just loop on the value, that's the reason I've been lazy 😅